core/consensus/babe/src/lib.rs

```diff
@@ -357,13 +372,13 @@ impl<Hash, H, B, C, E, I, Error, SO> SlotWorker<B> for BabeWorker<C, E, I, SO> w
 			}
 
 			let (header, body) = b.deconstruct();
-			let pre_digest: Result<BabePreDigest, String> = find_pre_digest::<B>(&header);
-			if let Err(e) = pre_digest {
-				error!(target: "babe", "FATAL ERROR: Invalid pre-digest: {}!", e);
-				return
-			} else {
-				trace!(target: "babe", "Got correct number of seals.  Good!")
-			};
+			// let pre_digest = find_pre_digest::<B>(&header);
```

Comment:
Needs updating.

Reply:
I will just delete this code.  It was strictly for debugging.